_content/essays/limits-of-desciption_geoff.md

```diff
@@ -0,0 +1,16 @@
+---
+title: "The Limits of Description"
+authors:
+  - geoff
+external_url: "https://www.dhammatalks.org/books/uncollected/NotSelfRevisited.html"
+drive_links:
+  - "https://drive.google.com/file/d/1JyxdPd6OK9NuetaBr4HKf9lVpnuEK9cc/view?usp=sharing"
+course: emptiness
+tags:
+  - philosophy
+  - view
+year: 2017
+pages: 16
+---
+
+In this essay, Ṭhānissaro Bhikkhu discusses a debate on "not self" as either a linguistic strategy or as an ontology. Ṭhānissaro Bhikkhu objectively presents his view and the opposing view of Venerable Bhikkhu Bodhi.
```

Comment:
Go ahead and link to Bhikkhu Bodhi's essay that this one is a response to:

https://buddhistuniversity.net/content/excerpts/anatta-as-ontology_bodhi